WalletWasabi/AssemblyInfo.cs

```diff
@@ -1,3 +1,6 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("WalletWasabi.Tests")]
+
+// For moq access to internals of this project.
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
Can you elaborate on this?

Reply:
I searched why this is required but is it safe? Can someone(Moq) manipulate our internal structures in library or gui code? How does this work?

Reply:
My answer would be as follows:

* You have only dependencies you trust in production code
* Tests are not in the final binary

If there is malicious code, then it does not really matter if you use `public` or `internal` because you can change it via reflection. Anyway, I think you are safe because you use only dependencies you trust and the attribute does not add any new dependencies to the code. 

@lontivero Could you please double-check that what I say is correct?

Anyway, if anyone knows how to get rid of this, it would be great. If Lucas finds this incorrect, I'll try to research it more and find a different solution.

Reply:
Yes, what you say is correct.